db/migrate/20200713194204_add_automated_priority_case_distribution_to_organization.rb

```diff
@@ -0,0 +1,5 @@
+class AddAutomatedPriorityCaseDistributionToOrganization < ActiveRecord::Migration[5.2]
+  def change
+    add_column :organizations, :automated_priority_case_distribution, :boolean, comment: "Whether a JudgeTeam is currently available for automatically pushed priority cases"
```

Comment:
My attempt to reduce words without losing meaning:
```suggestion
    add_column :organizations, :accepts_pushed_priority_cases, :boolean, comment: "Whether a JudgeTeam accepts automatically pushed priority cases"
```
We use [`priority_push`](https://github.com/department-of-veterans-affairs/caseflow/blob/master/db/migrate/20200713173228_add_priority_to_distributions.rb#L3) to describe distributions, so I'm trying to use it here. An alternative is `:accepts_priority_pushed_cases` or use  `accepts priority_pushed automatic case distributions` in the comment to enable finding the field while grepping.

Reply:
:tada: I like :accepts_priority_pushed_cases for the grep synergy!